x-pack/plugins/siem/public/components/filters_global/filters_global.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+import React from 'react';
+import { Sticky } from 'react-sticky';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { SuperDatePicker } from '../super_date_picker';
+
+const offsetChrome = 49;
+const gutterTimeline = '70px'; // Temporary until timeline is moved - MichaelMarcialis
+
+const Aside = styled.aside<{ isSticky?: boolean }>`
+  background: ${({ theme }) =>
+    theme.darkMode ? euiDarkVars.euiColorEmptyShade : euiLightVars.euiColorEmptyShade};
+  border-bottom: ${({ theme }) =>
+    theme.darkMode ? euiDarkVars.euiBorderThin : euiLightVars.euiBorderThin};
```

Comment:
Just a heads up, but the styled-components `<ThemeProvider>` is [configured](https://github.com/elastic/kibana/blob/master/x-pack/plugins/siem/public/apps/start_app.tsx#L34) with the currently selected `euiVars` on the `props.theme.eui` object, so these two could be simplified to just:

``` ts
background: ${({ theme }) => theme.eui.euiColorEmptyShade};
border-bottom: ${({ theme }) => theme.eui.euiBorderThin};
```

The props below could use the same technique and you shouldn't have to import `euiDarkVars` or `euiLightVars` :)

Reply:
👍